xml/Microsoft.Build.Tasks/Copy.xml

```diff
@@ -576,9 +564,7 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>
-            Create Hard Links for the copied files rather than copy the files if possible to do so
-            </summary>
+        <summary>Create Hard Links for the copied files rather than copy the files if possible to do so.</summary>
```

Comment:
```suggestion
        <summary>Gets or sets a value that indicates whether to use Hard Links for the copied files rather than copy the files, if it's possible to do so.</summary>
```